### ISSUE TITLE
altered checks for variable fonts: contour count

### ISSUE THREAD

**Issue by laerm0** [googlefonts/fontbakery#2284]
[redirect of https://github.com/googlefonts/fontbakery/issues/2047]

Variable fonts often have a unusual/unpredictable number of contours. I don't think fontbakery needs to check this for variable fonts.

**Comment by felipesanches** [comment]
Is this about **com.google.fonts/check/153** `Check if each glyph has the recommended amount of contours.` ?

If so, it is already setup so that it won't run on variable fonts anymore:
![screenshot at 2019-01-14 17 19 10](https://user-images.githubusercontent.com/213676/51135152-4a44d580-1831-11e9-9269-87522c01ec51.png)

@laerm0, are there other checks that you are thinking of here?

**Comment by laerm0** [comment]
Yeah, I'm actually checking my notes on this and it looks like you've already changed a lot of these. :) Feel free to close this. I'll comment on the source issue as well.
